src/gui/options_banlist.cpp

```diff
@@ -31,10 +31,20 @@
 #include "ui_options_banlist.h"
 
 #include <QMessageBox>
+#include <QHostAddress>
 #include <libtorrent/address.hpp>
 
 #include "base/bittorrent/session.h"
 
+namespace
+{
+    inline bool isValidIP(const QString &ip)
+    {
+        QHostAddress addr;
```

Comment:
"0.0.0.0" may not point to any host eligible for banning. Why one would want to ban it?